tests/data/mlip_fine_tune.yml

```diff
@@ -13,13 +13,13 @@ stress_weight: 100.0
 stress_key: dft_stress
 energy_key: dft_energy
 forces_key: dft_forces
-compute_stress: True
-compute_forces: True
+compute_stress: False
+compute_forces: False
```

Comment:
i woudl keep the forces, the rest I agree... also for testing will be useful to provide some valid file rather than a fraction